docs/docs/migration-guide.mdx

```diff
@@ -57,34 +57,59 @@ deprecated:
 - [Two-Stage-Fallback Policy](./policies.mdx#two-stage-fallback-policy)
 - [Form Policy](./policies.mdx#form-policy)
 
-#### Migrating from the Mapping Policy
+:::tip Automatic migration
+Rasa Open Source can take care of automatically migrating the Mapping Policy,
```

Comment:
should we add links to their deprecated pendants in the policy docs?

Reply:
the links are in the list above. I don't think it adds much TBH, and rather clutters the callout.